Dockerfile

```diff
@@ -18,6 +18,14 @@ COPY . .
 ENV CGO_ENABLED=1
 ENV GOOS=linux
 
+# Setup a directory for your certificates
+RUN mkdir /ssl
+
+# Generate a self-signed SSL certificate
+RUN openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes \
+  -keyout /ssl/tls.key -out /ssl/tls.crt \
+  -subj "/C=CN/ST=GD/L=SZ/O=ssv, Inc./CN=*.ssv.com"
```

Comment:
@MatusKysel do we have to add domain name? if so I would keep it as example.com or ssvlabs.io not ssv.com or anything else

Reply:
@MatusKysel @lior-blox 
we should understand what this means
is it a vulnerability if someone owns the domain we use here?

Reply:
We don't own ssv.com, if we need to have a domain then ssvlabs.io is the correct one to use

Reply:
@MatusKysel is there a way to not include any values at all? are these required? and if so, what are the defaults when you generate a self-signed certificate in the many tools that do so? (maybe we can copy them)